sdk/vc-data/package.json

```diff
@@ -20,7 +20,7 @@
     "build": "tsdx build --tsconfig ./tsconfig.build.json",
     "test": "tsdx test",
     "test:watch": "tsdx test --watch",
-    "lint": "tsdx lint src",
+    "lint": "tsdx lint src --fix",
```

Comment:
Why do we need this change? `npm run lint` is executed by CI; we want to make sure there are no linting errors, not to fix what is fixable?